packages/pwa/app/ssr.js

```diff
@@ -29,8 +32,9 @@ const app = createApp({
     manifestPath: 'static/manifest.json',
 
     // This is the value of the 'mobify' object from package.json
-    // provided by a webpack DefinePlugin
-    mobify: WEBPACK_PACKAGE_JSON_MOBIFY,
+    // provided by a webpack DefinePlugin. Alternatively the object can be defined
+    // in a configuration file in the `config` folder.
```

Comment:
i think the first half of this comment is no longer applicable. We no longer use the `DefinePlugin`, am i right?